ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/helpers/ArrayBuilder.java

```diff
@@ -37,7 +37,7 @@ public ArrayBuilder add(final Object value) {
   }
 
   public List<Object> build() {
-    return list;
+    return new ArrayList<>(list);
```

Comment:
As this is a builder, we should just make a deep-copy (not use `Collections.unmodifiableList()`)